resources/keywords/import_provider_account.resource

```diff
@@ -294,13 +301,16 @@ Provider Account Import Failure Using CLI
 Duplicate Provider Account Name Error Message Appears
     Element Should Be Visible    ${lblDuplProvAccName}
 
-User Navigates To Add ${databaseProvider} Topology Screen From Developer View
+User Navigates To Add Topology Screen From Developer View
+    [Arguments]     ${databaseProvider}     ${ns}=${operatorNamespace}      ${developer}=False
```

Comment:
${operatorNamespace} here relates to redhat-dbaas-operator?

Reply:
That's correct and if DBAASPOLICY.ENABLED on test-varaiables.yaml file set to "TRUE" then it would be overridden by the value on DBAASPOLICY.NAMESPACE

Reply:
OK got it.